virttest/storage.py

```diff
@@ -425,6 +455,36 @@ def auth_info_define_by_params(cls, image, params):
                 image, None, None, storage_type,
                 tls_creds=tls_creds, reconnect_delay=reconnect_delay
             ) if tls_creds or reconnect_delay else None
+        elif enable_curl:
+            # cookie data in a secure way, only for http/https
+            cookie_auth = cls(
+                image, params['curl_cookie_secret'],
+                params.get('curl_cookie_secret_format', 'raw'),
+                storage_type, data_type='cookie'
+            ) if (params['curl_protocol'] in ('http', 'https')
+                  and params.get('curl_cookie_secret')) else None
+
+            # sslverify, only for https/ftps
+            sslverify = params.get(
+                'curl_sslverify'
+            ) if params['curl_protocol'] in ('https', 'ftps') else None
```

Comment:
```suggestion
            sslverify = params.get('curl_sslverify')
```
Know from QEMU source code, the `sslverify` just only works on HTTPS and FTPS protocol.
So I think we don't need to check it again here because the caller should know whether to 
verify the SSL certificate.

Reply:
The beginning mind is doing an input validation, it's OK to let users to set it or not, and users should know that